docker-compose.yml

```diff
@@ -3,6 +3,6 @@ services:
   api:
     image: "veld/yagami"
     build: .
-    command: yarn build & yarn start
     env_file: .env
-    network_mode: "host"
\ No newline at end of file
+    ports:
+      - "${API_PORT}:${API_PORT}"
```

Comment:
running docker-compose on a fresh pull of this branch, yields
```
WARNING: The API_PORT variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.api.ports is invalid: Invalid port ":", should be [[remote_ip:]remote_port[-remote_port]:]port[/protocol]```

Reply:
set it with the .env file